dasharo-security/tpm-support.robot

```diff
@@ -20,22 +21,27 @@ Suite Setup         Run Keywords
 ...                     Prepare Test Suite
 ...                     AND
 ...                     Skip If    not ${TPM_SUPPORT}    TPM tests not supported
+...                     AND
+...                     Skip If    '${TPM_EXPECTED_VERSION}' == '0'    TPM Version unset
 Suite Teardown      Run Keyword
 ...                     Log Out And Close Connection
 
 
 *** Test Cases ***
 TPM001.001 TPM Support (firmware)
-    [Documentation]    This test aims to verify that the TPM is initialized
-    ...    correctly and the PCRs can be accessed from the firmware.
+    [Documentation]    This test aims to verify that the TPM is initialized,
+    ...    detected and logged correctly by FW via cbmem, directly in Ubuntu
     Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM001.001 not supported
     Power On
     Boot System Or From Connected Disk    ubuntu
     Login To Linux
     Switch To Root User
-    Get Cbmem From Cloud
-    ${out}=    Execute Command In Terminal    cbmem -L
-    Should Contain Any    ${out}    TPM2 log    TCPA log
+    Verify Presence Of TPM Via Sysfs
```

Comment:
These keywords in this order are repeated in almost all of the tests, perhaps it would make sense to extract them to a local keyword.